packages/sdk-client-v3/src/utils/byteLength.ts

```diff
@@ -1,8 +1,10 @@
-var Buffer = require('buffer/').Buffer
-
 export default function byteLength<T>(body: T): string {
-  if (body && (typeof body === 'string' || body instanceof Uint8Array)) {
-    return Buffer.byteLength(body).toString()
+  if (body && typeof body === 'string') {
+    return body.normalize('NFD').length.toString()
+  }
+
+  if (body && body instanceof Uint8Array) {
```

Comment:
I'm just curious, in which case we can get body as Uint8Array?

Reply:
For images, we have an image endpoint that will fail is this check if not done.

```ts
// Something like this...
...
  return apiRoot
    .withProjectKey({ projectKey })
    .products()
    .withId({ ID: '' })
    .images()
    .post({
      headers: {
        "Content-Type": "image/jpeg"
      },
      body: fileFromSync(path.resolve(__dirname, 'image.jpeg'), 'image/jpeg'),
      queryArgs: {
        filename: 'DKNY.jpeg'
      }
    })
    .execute();
}
...
```